buildenv/jenkins/perfPipeline.groovy

```diff
@@ -99,16 +102,29 @@ timestamps {
                                 // test
                                 testParams << string(name: "TEST_NUM", value: "TEST_NUM" + i.toString())
                                 def testRun = triggerJob("${item.BENCHMARK}", "${platform}", testParams, "test")
-                                aggregateLogs(testRun)
+                                aggregateLogs(testRun, testRuntimes)
 
                                 // baseline
                                 if (RUN_BASELINE) {
                                     baselineParams << string(name: "BASELINE_NUM", value: "BASELINE_NUM_" + i.toString())
                                     def baseRun = triggerJob("${item.BENCHMARK}", "${platform}", baselineParams, "baseline")
-                                    aggregateLogs(baseRun)
+                                    aggregateLogs(baseRun, baselineRuntimes)
                                 } else {
                                     echo "Skipping baseline run since RUN_BASELINE is set to false"
                                 }
+
+                                testStats = stats(testRuntimes)
+                                baselineStats = stats(baselineRuntimes)
+                                def score = (testStats.mean/baselineStats.mean) * 100
+
+                                echo "testRuntimes: ${testRuntimes}" 
+                                echo "baselineRuntimes: ${baselineRuntimes}"
+                                echo "score: ${score} %"
+
+                                if (score <= 98) {
```

Comment:
We should only set the build result at the end of `PERF_ITERATIONS` or `EXIT_EARLY` (https://github.com/adoptium/aqa-tests/issues/6330). For example:

```
if (i == PERF_ITERATIONS || ( EXIT_EARLY && i >= PERF_ITERATIONS * 0.8) ) {
   if (score <= 98) {
    ....
   } else {
       echo "Perf iteration completed. EXIT_EARLY: ${EXIT_EARLY}. PERF_ITERATIONS: ${PERF_ITERATIONS}. Actual iterations: ${i}."
       break
   }
}
```

Reply:
I missed `params`. It should be params.EXIT_EARLY.
Please test the PR after the change. Thanks.